app/services/fastboot.js

```diff
@@ -46,5 +46,8 @@ export default Ember.Service.extend({
 
   isFastBoot: computed(function() {
     return typeof FastBoot !== 'undefined';
-  })
+  }),
+  deferRendering(promise) {
```

Comment:
If we were to add a hook, where would we introduce it?

This is to handle the case that you have asynchrony outside of the normal hooks. as such, you may be doing this work inside a component.

Or are you suggesting that we make a new promise from the promise being passed into `deferRendering`